src/ProjectService.Data/UserRepository.cs

```diff
@@ -50,7 +69,20 @@ public void AddUsersToProject(IEnumerable<DbProjectUser> dbProjectUsers , Guid p
 
         public IEnumerable<DbProjectUser> Find(Guid userId)
         {
-            return _provider.ProjectsUsers.Where(x => x.UserId == userId);
+            return Find(new FindDbProjectsUserFilter { UserId = userId });
+        }
+
+        public IEnumerable<DbProjectUser> Find(FindDbProjectsUserFilter filter)
+        {
+            if (filter == null)
+            {
+                throw new ArgumentNullException(nameof(filter));
+            }
+
+            var dbProjectsUser = _provider.ProjectsUsers
+                .AsQueryable();
```

Comment:
add few tabs or write in previous line, it looks ugly (but it is only my opinion)

Reply:
done